apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -12,6 +12,7 @@ import { useSession } from "next-auth/react";
 import dynamic from "next/dynamic";
 import Head from "next/head";
 import { useRouter } from "next/router";
+import { HashLinkPageProps } from "pages/d/[link]/book";
```

Comment:
Do we mean to use an absolute import here?

Reply:
Oh, right. Updating this to be similar to `BookPageProps` & `TeamBookingPageProps`